ignite/contrib/metrics/mae.py

```diff
@@ -4,11 +4,25 @@
 
 
 class MaximumAbsoluteError(Metric):
```

Comment:
Could you please also rename the file `mae.py` -> `maximum_absolute_error.py` to have a coherence with others.

Reply:
Done!